### DISCUSSION TITLE
academy/node-js/scraping-urls-list-from-google-sheets

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#557] [Comments]
# academy/node-js/scraping-urls-list-from-google-sheets

Learn how to crawl a list of URLs specified in a Google Sheets document using one of the Apify web scraping actors.

https://docs.apify.com/academy/node-js/scraping-urls-list-from-google-sheets

**Comment by omcknightkittl** [comment]
I do not see anywhere in the Tiktok Webscrapers actor inputs to Link to remote text file.
